dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -208,13 +208,13 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml",
 		"integration/kubernetes/helm-chart/alluxio/templates/job/format-journal-job.yaml",
 		"integration/kubernetes/helm-chart/alluxio/templates/master/journal-pv.yaml",
-		"integration/kubernetes/helm-chart/alluxio/templates/master/journal-pvc.yaml",
 		"integration/kubernetes/helm-chart/alluxio/templates/master/service.yaml",
 		"integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml",
 		"integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml",
 		"integration/kubernetes/multiMaster-embeddedJournal/alluxio-configmap.yaml.template",
 		"integration/kubernetes/multiMaster-embeddedJournal/config.yaml",
 		"integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template",
+		"integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-journal-pv.yaml.template",
```

Comment:
should we consider removing all "pv" from templates? this way `helm install` does not attempt to create a pre-req

Reply:
Makes sense to me that we should remove the PVs and let the users handle this with instructions in docs. I agree.